pkg/assets/static/redirectpage.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en" class="pf-m-redhat-font">
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <title>OpenShift Developer Sandbox</title>
+      </head>
+      <body>
+      </body>
+      <script src="redirectpage.js"></script>
```

Comment:
Doesn't seem to be working...

This is what I tried: `{baseURL}/redirectpage.html?link="import"`
And I expect to get redirected to `{consoleURL}/import/ns/alexeykazakov-dev`
But the JS failed with:
```
redirectpage.js:19 Uncaught SyntaxError: Unexpected token 'export'
jquery.min.js:14 [Deprecation] Listener added for a 'DOMSubtreeModified' mutation event. Support for this event type has been removed, and this event will no longer be fired. See https://chromestatus.com/feature/5083947249172480 for more information.
add @ jquery.min.js:14
```